transport/grpc/outbound.go

```diff
@@ -330,6 +334,27 @@ func (o *Outbound) stream(
 	return tClientStream, nil
 }
 
+// Introspect implements introspection.IntrospectableOutbound interface.
+func (o *Outbound) Introspect() introspection.OutboundStatus {
```

Comment:
Note that this is identical to HTTP
https://github.com/yarpc/yarpc-go/blob/3b2ec4458cf8f631a98bc15b704698f415f67c72/transport/http/outbound.go#L581